### ISSUE TITLE
Link website on GitHub description

### ISSUE THREAD

**Issue by williamdes** [bluejekyll/trust-dns#1516]
Could you please link http://trust-dns.org/ if it is the official website on this repository in the same section as the tags ?
And have a working https ;) ?

Thank you for this project !

**Comment by bluejekyll** [comment]
Yes. I’ve been lazy and need to update that site. I want to make it the primary landing site with docs, but haven’t gotten around to making the Zola project for it yet…

**Comment by williamdes** [comment]
> Yes. I’ve been lazy and need to update that site. I want to make it the primary landing site with docs, but haven’t gotten around to making the Zola project for it yet…

Maybe I can help out ?
cc @Guileas

**Comment by williamdes** [comment]
Should the old website redirect to the new one ?

**Comment by williamdes** [comment]
/refresh
what should be done now 🤔 ?

**Comment by bluejekyll** [comment]
Thanks for the ping on this. I’ll see about trulying to update it.

**Comment by bluejekyll** [comment]
I'm trying to figure out how to require and map the DNS name for trust-dns.org from the original GitHub pages for the project, but since I moved the original project that the trust-dns.org pages site was attached to, I can't recover access to the pages config for trust-dns. I've created a new project in my personal account for trust-dns, and published a new site that redirects to hickory-dns: https://github.com/bluejekyll/trust-dns and then the pages site: https://bluejekyll.github.io/trust-dns/

The redirection works properly. I've done what GitHub says to do here, which is to verify the domain, https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/verifying-your-custom-domain-for-github-pages#verifying-a-domain-that-is-already-taken

It might take a week, `note that the process to release the domain from its current location will take 7 days to complete`. So maybe we have to wait a week?

**Comment by williamdes** [comment]
@bluejekyll where are we now with this one ?
